Library/Homebrew/dependency.rb

```diff
@@ -4,8 +4,6 @@
 require "dependable"
 
 # A dependency on another Homebrew formula.
-#
-# @api private
 class Dependency
```

Comment:
This class is used in Homebrew/test-bot, I believe.

Reply:
Couln't find it in `test-bot`.